### PR TITLE
feature-8800: Revert UI for 2 field Consent Form Field & Consent Refu…

### LINKED ISSUE #8800: Make custom form question translatable by the organizer
Forms can be translated into other languages, but custom form question cannot. Therefore please add this feature to the attendee form.

* Add a button "Add translation"
* Add a dropdown menu with a list of languages when the user clicks
* After the user has chosen the language show a form field to add the question (show which languages was translated)
* When the user changes the interface language show the correct translation.

Before

Screenshot from 2023-06-16 12-02-55

After
Screenshot from 2023-06-16 11-50-19

### PR DESCRIPTION
**Pull request by lthanhhieu**
…nd Policy

Fixes #8800 

#### Short description of what this resolves:
- Make custom form question translatable by the organizer

#### Changes proposed in this pull request:
- Revert UI for 2 field Consent Form Field and Consent Refund Policy

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by cweitat** [comment]
@lthanhhieu the translations for the text are not included yet.

File: app/templates/components/forms/orders/order-form.hbs [hunk 1]
```diff
@@ -230,7 +230,25 @@
...
+                    @checked={{get holder field.identifierPath}}         
+                    @name={{if field.isRequired (concat field.fieldIdentifier "_required_" index) (concat field.fieldIdentifier "_" index)}}       
+                    @onChange={{action (mut (get holder field.identifierPath))}} />
+                    <span>{{t 'I agree to the terms of the'}} </span>
+                    <a href="https://foundation.wikimedia.org/wiki/Policy:Universal_Code_of_Conduct">{{t 'Universal Code of Conduct'}}</a>
+                    <span> {{t 'and the'}} </span>
+                    <a href="https://foundation.wikimedia.org/wiki/Policy:Friendly_space_policy">{{t 'Friendly Space Policy'}}</a>
+                    <span> .</span>
+                {{else if (eq field.type 'checkbox')}}
+                  {{#if (and (not-eq field.fieldIdentifier "language_form_1") (not-eq field.fieldIdentifier "language_form_2"))}}
```
Comment (cweitat) [inline app/templates/components/forms/orders/order-form.hbs:245]: is there a better naming convention other than language_form_1?

**Review by cweitat — CHANGES_REQUESTED** [review]
https://github.com/fossasia/open-event-frontend/issues/8800#issuecomment-1612028211

this issue to be fixed as well

**Comment by lthanhhieu** [comment]
> 
Hi @cweitat
I fixed this issue. Please help me to check it
https://github.com/fossasia/open-event-frontend/pull/8863/commits/098b0d06db5d90730430cd5547f4d50a6116b865

**Comment by lthanhhieu** [comment]
> [#8800 (comment)](https://github.com/fossasia/open-event-frontend/issues/8800#issuecomment-1612028211)
> 
> this issue to be fixed as well

**Comment by cweitat** [comment]
> > 
> 
> Hi @cweitat I fixed this issue. Please help me to check it [098b0d0](https://github.com/fossasia/open-event-frontend/commit/098b0d06db5d90730430cd5547f4d50a6116b865)

Please see my review comment. Thanks

File: app/templates/components/forms/orders/order-form.hbs (hunk 1 above)
Comment (lthanhhieu) [reply app/templates/components/forms/orders/order-form.hbs:245]: Hi @cweitat.

Sorry for this name. I implemented this name on ticket [#8788](https://github.com/fossasia/open-event-frontend/issues/8788). So I will re-open this ticket and convert new name

Thanks

**Comment by cweitat** [comment]
@lthanhhieu please add screenshot thanks

**Comment by khangon** [comment]
Hi @cweitat,
Please help me check screenshot.
Thank you,
image

**Pull request merged**